### ISSUE TITLE
Asylum Teaser example dissapeared.

### ISSUE THREAD

**Issue by inca2000** [Senscape/Dagon#156]
http://www.facethehorror.com/files/Asylum_Teaser_Unix.zip does not exist.
Neither does https://talk.senscape.io/asylum-teaser

**Comment by bd339** [comment]
Troublesome indeed. [The teaser repo](https://github.com/Senscape/Asylum-Teaser) only has the scripts and I don't have a copy of the Asylum teaser any more. You can download Linux binaries for the engine itself [here](https://github.com/Senscape/Dagon/releases/tag/v0.6.5-alpha). Mind you that for running the Asylum teaser I'm pretty sure the version I linked you to is the latest version you can run. Quite a bit has happened in Dagon since 0.6.5a so it is no longer compatible with the teaser. Then you just need the assets... heh, sorry I don't know where to get those :sweat_smile:

**Comment by agustincordes** [comment]
Yeah, I'm in the process of redoing the whole site, so none of the older pages are available. We should be relaunching fairly soon, but I'm not sure if I'm going to do a dedicated page for the old teaser -- however, I may look into uploading those old files to itch.io, maybe.

**Comment by inca2000** [comment]
It's mostly so I'd have a better idea of the Dagon workflow. I guess I'll just keep at it blind. Curious to see the redo!

**Comment by bd339** [comment]
Do you have Serena installed? That game is also made with Dagon and in the installation folder you can see how it's made by browsing the scripts.

**Comment by darealshinji** [comment]
You can download the teaser from IndieDB: http://www.indiedb.com/games/asylum-face-the-horror/downloads/asylum-interactive-teaser-ubuntu

Use the script provided here to re-arrange the files, so they work with newer versions of the engine: https://github.com/Senscape/Asylum-Teaser/blob/master/download-game-content.sh

Here you can get pre-compiled binaries for Linux, if you need them: https://drive.google.com/file/d/0BxClEVSW8QykRVdqTDhuWURoaFU/view?usp=sharing
They're somewhat portable. Make sure you know how to use `LD_LIBRARY_PATH`.
